config/travellers/pajowu.yaml

```diff
@@ -0,0 +1,21 @@
+name: "pajowu"
+profile:
+  web: ["https://pajowu.de", "https://einhorn.space"]
```

Comment:
I am about 80% sure that this won't work, but I'd really like to see it fail, so let's give it a shot! :dancer: